edge_intelligence_models/eval-model.py

```diff
@@ -1,90 +0,0 @@
-import tensorflow as tf
```

Comment:
Change file permission to 644

Reply:
All files which are being shown outdated moved to examples/edge_intelligence/trained_models